src/Abp.AspNetCore/AspNetCore/AbpApplicationBuilderExtensions.cs

```diff
@@ -106,5 +106,23 @@ public static void UseAbpRequestLocalization(this IApplicationBuilder app, Actio
                 app.UseRequestLocalization(options);
             }
         }
+
+        public static void AddSecurityHeaders(this IApplicationBuilder app)
```

Comment:
1. Rename to UseSecurityHeaders, because it's a convention to begin with "Use" for middlewares. "Add" prefix is for adding services to DI (IServiceCollection).
2. Add headers after ``await next();``, thus we can be sure that next middlewares will not clear them. Also add them if not exists.